qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -27,38 +27,35 @@
 
 
 class T2Ramsey(BaseExperiment):
+    r"""T2 Ramsey Experiment.
 
-    r"""
-    T2Ramsey Experiment
+    # section: overview
 
-    Overview
         This experiment is used to estimate two properties for a single qubit:
         T2* and Ramsey frequency.
-        This experiment consists of a series of circuits of the form
 
-    .. parsed-literal::
+        See `Qiskit Textbook <https://qiskit.org/textbook/ch-quantum-hardware/\
+        calibrating-qubits-pulse.html>`_  for a more detailed explanation on
+        these properties.
 
-           ┌───┐┌──────────────┐┌──────┐ ░ ┌───┐ ░ ┌─┐
-      q_0: ┤ H ├┤   DELAY(t)   ├┤ RZ(λ)├─░─┤ H ├─░─┤M├
-           └───┘└──────────────┘└──────┘ ░ └───┘ ░ └╥┘
-      c: 1/═════════════════════════════════════════╩═
-                                                    0
+        This experiment consists of a series of circuits of the form
 
-    for each *t* from the specified delay times, and where
-    :math:`\lambda =2 \pi \times {osc\_freq}`,
-    and the delays are specified by the user.
-    The circuits are run on the device or on a simulator backend.
-    Results are analysed in the class T2RamseyAnalysis.
+        .. parsed-literal::
 
-    Analysis Class
-        :class:`~qiskit.experiments.characterization.T2RamseyAnalysis`
+                 ┌───┐┌──────────────┐┌──────┐ ░ ┌───┐ ░ ┌─┐
+            q_0: ┤ H ├┤   DELAY(t)   ├┤ P(λ) ├─░─┤ H ├─░─┤M├
+                 └───┘└──────────────┘└──────┘ ░ └───┘ ░ └╥┘
+            c: 1/═════════════════════════════════════════╩═
+                                                        0
 
-    Analysis Options
+        for each *t* from the specified delay times, where
+        :math:`\lambda =2 \pi \times {osc\_freq}`,
+        and the delays are specified by the user.
+        The circuits are run on the device or on a simulator backend.
+        Results are analysed in the class T2RamseyAnalysis.
```

Comment:
You can remove the last line. A similar sentence is going to be generated automatically, with a link to the `T2RamseyAnalysis` class. If you still prefer to keep the sentence, then it's better to provide a link, or at least encapsulate `T2RamseyAnalysis` with ``.

Reply:
Removed the last sentence.